test/mailers/notifier_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+
+class NotifierTest < ActionMailer::TestCase
+
+  def setup
+    recipients = User.all.map {|user| user.email}
```

Comment:
@vlasiak, Her you could use `pluck` as well. Ideally, in order to test notifier, I would mock anything that hits DB. I will explain in details soon.